docs/LOGGING.md

```diff
@@ -14,11 +14,11 @@ The current log pipeline can be vertically split into the following three stages
 
 ## Log Generation
 
-All containers will now log directly to stdout, which are then picked up by containerd. Containerd then forwards the logs received to memlogd service that maintains a circular buffer of fixed size for storing log messages. Memlogd provides a way (via unix domain sockets) to read the logs present in it’s buffer and also stream the incoming logs. EVE currently uses imemlogd plugin for rsyslogd (developed internally) to stream logs from memlogd service.
+All containers will now log directly to stdout, which are then picked up by containerd. Containerd then forwards the logs received to memlogd service that maintains a circular buffer of fixed size (5000 messages limit and 8192 bytes limit per message size) for storing log messages. Memlogd provides a way (via unix domain sockets) to read the logs present in it’s buffer and also stream the incoming logs. Log from Pillar services are also forwarded into the memlogd circular buffers.
```

Comment:
I think better terminology is to call pillar a container (services are things like baseosmgr). Thus you don't need the last sentence in this paragraph.

Reply:
ok.